### DISCUSSION TITLE
Ingredients that reference other recipes

### DISCUSSION THREAD

**Discussion by cashpw** [mealie-recipes/mealie#858] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

Some ingredients for a recipe are themselves products of another recipe. As an example: a recipe for a cake which calls for a particular frosting for which a recipe exists.

**(Optional): Suggest A Solution**

The currently-supported method to provide a link from (using the above example) the cake recipe to the frosting recipe, as far as I can tell, is to include the link in the `Notes` section of the cake recipe. It would be nice to be able to have the link present in the list of ingredients such that clicking on "Frosting" would take you to the frosting recipe.

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Recipe text should support markdown syntax so you can embed a link to a recipe 

```md
[My Recipe](https://mealie.mydomain.com/recipes/my-recipe)
```

Which renders a link like below...

[My Recipe](https://mealie.mydomain.com/recipes/my-recipe)

---

Doing a database-level link would require a lot of additional work to keep the imports/exports working. I think the markdown link is a good enough solution, but others have brought up the same concern. .

**Reply by cashpw** [reply]
Markdown is a fine solution. I didn't realize that those fields supported markdown.
